validator/src/main/resources/expected-data-template/dotnet/ec2/asg/aws-sdk-call-trace.mustache

```diff
@@ -0,0 +1,69 @@
+[{
+  "name": "^{{serviceName}}$",
+  "http": {
+    "request": {
+      "url": "^{{endpoint}}/aws-sdk-call$",
+      "method": "^GET$"
+    },
+    "response": {
+      "status": "^500$"
+    }
+  },
+  "aws": {
+    "account_id": "^{{accountId}}$"
+  },
+  "annotations": {
+    "aws.local.service": "^{{serviceName}}$",
+    "aws.local.operation": "^GET aws-sdk-call$",
+    "aws.local.environment": "^ec2:{{platformInfo}}$"
+  },
+  "metadata": {
+    "default": {
+      "EC2.AutoScalingGroup": "^{{platformInfo}}$",
+      "EC2.InstanceId": "^{{instanceId}}$",
+      "PlatformType": "^AWS::EC2$",
+      "otel.resource.ec2.tag.aws:autoscaling:groupName": "^{{platformInfo}}$",
+      "otel.resource.host.id": "^{{instanceId}}$",
+      "otel.resource.host.image.id": "^{{instanceAmi}}$",
+      "otel.resource.host.type": "^t3.micro$",
+      "otel.resource.host.name": "^{{privateDnsName}}$",
+      "aws.span.kind": "^LOCAL_ROOT$"
+    }
+  },
+  "subsegments": [
+    {
+      "name": "^S3$",
+      "http": {
+        "response": {
+          "status": "^404$"
```

Comment:
Don't need to assert the status, it's not something that will always be the same because we can eventually get throttled or otherwise.

Reply:
OK removed from ASG and Default, pending test

Reply:
You didn't remove status